reana_server/reana_admin/main.py

```diff
@@ -639,3 +640,61 @@ def set_default_quota_limit(ctx):
             resource_type=resource.type_.name,
             limit=DEFAULT_QUOTA_LIMITS.get(resource.type_.name),
         )
+
+
+@reana_admin.command("consume-queue")
+@click.option(
+    "--queue-name",
+    "-q",
+    required=True,
+    type=str,
+    help="Name of the queue that will be consumed, e.g workflow-submissions",
+)
+@click.option(
+    "--key",
+    "-k",
+    type=str,
+    help="Key that will be used to get the id value from the message body, e.g workflow_name_or_id",
```

Comment:
What about `Key of the property that will be used to filter the messages in the queue, e.g. workflow_name_or_id`?